src/liballoc/string.rs

```diff
@@ -621,14 +622,15 @@ impl String {
     }
 
     /// Decode a UTF-16 encoded slice `v` into a `String`, replacing
-    /// invalid data with the replacement character (U+FFFD).
+    /// invalid data with [the replacement character (`U+FFFD`)][U+FFFD].
```

Comment:
was this missed in the push to `U+FFFD REPLACEMENT CHARACTER` everywhere (the above comment)?

Reply:
Not missed, I intentionally didn't change this instance. Not sure I see the value in making all of these cases exactly identical